inst/WORDLIST

```diff
@@ -7,15 +6,12 @@ TLG
 UI
 UIs
 UX
-analysing
 cloneable
 customizable
-dropdown
```

Comment:
@kartikeyakirar small change to vignettes to remove 2 unecessary entries from wordlist